File: RB101_programming_foundations/lesson_5/5_practice_problems/ex_07.rb
# Given this code, what would be the final values of a and b? Try to work this out without running the code.

a = 2
b = [5, 8]
arr = [a, b] # => [2, [5, 8]]

p arr[0] += 2 # => [4, [5, 8]]
p arr[1][0] -= a # => [4, [3, 8]]

p a # => a = 2
p b # => B = [3, 8]

File: JS101_programming_foundations/lesson_5/practice_problems/ex_07.js
// Given the following code, what will the final values of a and b be ? Try to answer without running the code.

let a = 2;
let b = [5, 8];
let arr = [a, b]; // [2, [5, 8]]

arr[0] += 2; // [4, [5, 8]]
arr[1][0] -= a; // [4, [3, 8]]

console.log(a);
console.log(b);

// a = 2, b = [[3, 8]]
